e2e_tests/test_advanced_search_page.py

```diff
@@ -1,13 +1,131 @@
+import re
+
+import pytest
 from playwright.sync_api import Page, expect
 
 from .utils.assertions import assert_is_accessible
 
 
-def test_advanced_search_page(page: Page):
+def query_input(page: Page):
+    return page.locator("#search_form")
+
+
+def submit_button(page: Page):
+    return page.get_by_role("button", name="Submit")
+
+
+def update_filters_button(page: Page):
+    return page.get_by_role("button", name="Update filters")
+
+
+court_filters = [
```

Comment:
This feels like it could be fragile; can we derive this from the utils?

Reply:
I would love to do that, but the E2E docker app currently isn't set up with any of the dependencies from the main app. I had a go at getting this docker image to match the main app but failed before. I can try to do this again as a follow up PR if that works? Or if someone else wants to have a go at getting the utils available here rather than me flailing around that would be appreciated 😄

Reply:
Nah, we can fix it later.